run/django/.gcloudignore

```diff
@@ -0,0 +1,20 @@
+# This file specifies files that are *not* uploaded to Google Cloud Platform
```

Comment:
not sure if y'all normally commit gcloudignores for cloud run, if you do, cool, if not, you can remove this.

Reply:
The user is directed to build the container from this directory, so in theory there should be none of the files listed locally available to accidentally add to the zip sent to Cloud build and inflate the size of the bundle. I've removed.